### PR TITLE
Second milestone

### PR DESCRIPTION
**Pull request by edwinmoradian90**
## Changes made
- Add components folder
- Add Button, ButtonPanel, Display, and App to components
- Add proptypes
- Add button names for all buttons
- Display buttons on button panel
- Display '0' as default result

### CODE REVIEW

File: src/components/Display.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+};
+
+export default Display;
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
+
+Display.defaultProps = {
+  result: '',
```
Comment (MauricioRobayo) [inline src/components/Display.jsx:20]: Please adjust the default value according to the requirements 😄 .

File: src/components/ButtonPanel.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      <Button name={buttons[13]} />
+      <Button name={buttons[14]} />
+      <Button name={buttons[15]} />
+    </div>
+    <div className="groupFive">
+      <Button name={buttons[16]} />
+      <Button name={buttons[17]} />
+      <Button name={buttons[18]} />
+    </div>
+  </div>
```
Comment (MauricioRobayo) [inline src/components/ButtonPanel.jsx:37]: OPTIONAL: Nice! If there were a way to map through the array to build this, we could avoid so much duplicate code, we might need to modify the array a little bit maybe 🤔 .

File: src/components/Display.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+export default Display;
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
+
+Display.defaultProps = {
+  result: '0',
+};
```
Comment (shloch) [inline src/components/Display.jsx:21]: good

**Comment by edwinmoradian90** [comment]
Thanks!

**Pull request merged**